packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -415,11 +415,11 @@ abstract class ScrollView extends StatelessWidget {
 ///
 /// This semantic index is not necessarily the same as the index of the widget in
 /// the scrollable, because some widgets may not contribute semantic
-/// information. Consider a [new ListView.separated()]: every other widget is a
+/// information. Consider a [ListView.separated()]: every other widget is a
```

Comment:
while you're in here: remove the `()` here and below?

Reply:
What about this one?

Reply:
Oh, didn't see this comment, my apologies.

Reply:
done